SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -208,6 +208,9 @@ fi
 ln -sf linux-%{uname_r}.cfg /boot/mariner-mshv.cfg
 %grub2_post
 
+%posttrans
+systemctl mask kdump.service >/dev/null 2>&1 || :
```

Comment:
Should this really be masked at the kernel-mshv RPM transaction level? Maybe this masking is better done at the image composition level?

Reply:
Given your above comment about not changing crashkernel= reservation and this, I think I will close this PR and instead do the masking in AgentBaker.